web/README.md

```diff
@@ -1,137 +1,50 @@
-
-If you need an usual global selector, use:
-
-```
-:global(.my-selector)
-```
-
-Read more on CSS Modules here: https://github.com/css-modules/css-modules
-
-Read more on SCSS here: https://sass-lang.com/
-
-## SVG icons
-
-SVG files can be imported like this:
-
-```
-import { ReactComponent as myIcon } from 'icons/myIcon.svg'
-```
-
-After this, myIcon.svg will be included in SVG sprite as a symbol, and myIcon variable will contain and object with meta-information about the SVG file, like { id, url, viewBox } or like React.SFC.
-
-[comment]: <> (## Working the .NET/Java server)
-
-[comment]: <> (By default, the project is configured to run without the server &#40;like .NET/Java-based REST APIs&#41;.)
-
-[comment]: <> (Usually, back-end project serves static assets as well as it's REST APIs. Considering this, when you'll get back-end server running, you need to change some settings:)
-
-[comment]: <> (- make sure your server can serve static files from /built path.)
-
-[comment]: <> (- the app uses Single Page Application approach, so routing is done at client-side. To support this, you'll need to tweak your server-side routing so any unknown path &#40;like /home&#41; serves the /built/index.html file content)
-
-[comment]: <> (- on developers machines, setup the server to run on some spare port &#40;like 44301&#41;)
-
-[comment]: <> (- Back-end developers can use "yarn build" after getting the source, to get recent version of UI scripts)
-
-[comment]: <> (- On your build/CI script use "yarn prod" command to build the bundle.js.)
-
-# Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn about UII, visit [UUI documentation website](https://uui.epam.com/)
+## Local Development
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+To work in local environment you need to start up the **_local backend_** and configure env variables for correct connection.
+For that you need to copy content of `.env.example` file to `.env.local` and specify provided variables.
 
 ## Project Structure Overview
 
 <img src='./docs/structure.png' />
 
-## Local Development
-
-To work in local environment you need to start up the **_local backend_** and configure env variables for correct connection.
-For that you need to copy content of `.env.example` file to `.env.local` and specify provided variables.
+# Learn More
 
-## Test 
-Libraries for testing:
-- React testing library 
-- Jest 
\ No newline at end of file
+This project is using CRA, you can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).\
+To learn about UII, visit [UUI documentation website](https://uui.epam.com/).\
+To learn React, check out the [React documentation](https://reactjs.org/).
```

Comment:
new link for v17 - [legacy.reactjs.org](https://legacy.reactjs.org/docs/hello-world.html)
for v18+ - [react.dev](https://react.dev/)